drain3/template_miner.py

```diff
@@ -170,24 +171,93 @@ def match(self, log_message: str, full_search_strategy="never") -> LogCluster:
         matched_cluster = self.drain.match(masked_content, full_search_strategy)
         return matched_cluster
 
-    def get_parameter_list(self, log_template: str, content: str):
+    def get_parameter_list(self, log_template: str, content: str, exact_matching: bool = True) -> [str]:
+        """
+        Extract parameters from a log message according to a provided template.
```

Comment:
according to a provided template that was generated by Drain3 by calling `add_log_message()`